tests/test_gradient.py

```diff
@@ -98,7 +104,7 @@ def test_gradient_equivalence(self, shape, kernel, space_order, preset, nbl, dty
         s = model.grid.stepping_dim.spacing
 
         eqn_adj = iso_stencil(v, model, kernel, forward=False)
-        receivers = rec.inject(field=v.backward, expr=rec * s**2 / m)
+        receivers = rec_true.inject(field=v.backward, expr=rec_true * s**2 / m_true)
```

Comment:
m0

Reply:
Does it matter which `m` was used to construct the expression, when all `op.apply()` calls are explicit about `m_true` and `m0`?

Reply:
Makes it more readable what is what

Reply:
Done